bin/metadata.js

```diff
@@ -32,8 +33,10 @@ async function main(prid, owner, repo, logger) {
   if (!process.stdout.isTTY) {
     process.stdout.write(`${metadata}${EOL}`);
   }
+
   const checker = new PRChecker(logger, data);
   checker.checkAll();
+  commitAfterReview(data.commits, data.reviews, logger);
```

Comment:
This seems like it should be called from the checker?

Reply:
@apapirovski Can you explain more about it.

Reply:
I just mean that it's conceptually similar in its purpose to the things that are being checked within `checkAll`, so this could be called from there rather than from `bin/metadata.js` and then it could also exist as a method of that `PRChecker` class and have access to the data, instead of needing it passed in.

Reply:
I agree, this can be implemented as `PRChcker.checkCommitAfterReview`